server/routes/api/businesses.js

```diff
@@ -29,14 +27,15 @@ router.get("/getThree", (req, res) => {
 });
 
 
-router.post("/byCategory", (req, res) => {
+router.get("/getByCategory/:category", (req, res) => {
   Business.findAll({
-    where: {
-      type: req.body.category,
-    },
-  }).then((Businesses) => res.json(Businesses))
+    where: { category: req.params.category },
+  })
+    .then((Businesses) => res.json(Businesses))
     .catch((err) =>
-      res.status(404).json({ noBusinessesFound: "No Businesses found in the DB" })
+      res
```

Comment:
you don't need to change this here but in general it's better to handle this with a 400 status code unless you are sure that no businesses were found. In our case finding no businesses isn't really an error